### PR TITLE
show min and max amounts in `loop out -h`

### PR DESCRIPTION
**Pull request by andrewn6**
#### Pull Request Checklist
- [] Update `release_notes.md` if your PR contains major features, breaking changes or bugfixes

issue #342

### CODE REVIEW

**Comment by sputn1ck** [comment]
I don't think it makes sense to hardcode the amounts in the help section, as the amounts are decided on the loop server. If you type `loop terms` you will see the amounts, as @guggero suggested in #342.

Maybe it makes sense to hint at `loop terms` in the help command.

File: cmd/loop/loopout.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ var loopOutCommand = cli.Command{
 		},
 		cli.Uint64Flag{
 			Name:  "amt",
-			Usage: "the amount in satoshis to loop out",
+			Usage: "run the command loop terms to get the minimum & maximum amount on the server. ",
```
Comment (GeorgeTsagk) [inline cmd/loop/loopout.go:44]: > run the command loop terms to get the minimum & maximum amount on the server.

better rephrase as "the amount in satoshis to loop out, consult the loop terms command to get the minimum & maximum loop amounts supported by the server."

Comment (anddddrew) [reply cmd/loop/loopout.go]: updated it to this 

https://github.com/lightninglabs/loop/pull/543/commits/155ccd5ce6f72ec9085dce4742df988602f7cdc9

Comment (anddddrew) [reply cmd/loop/loopout.go]: I think its in one commit now?

File: cmd/loop/loopout.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ var loopOutCommand = cli.Command{
 		},
 		cli.Uint64Flag{
 			Name:  "amt",
-			Usage: "the amount in satoshis to loop out",
+			Usage: "the amount in satoshis to loop out, consult the loop terms command to get the minimum & maximum loop amounts supported by the server.",
```
Comment (sputn1ck) [inline cmd/loop/loopout.go:44]: We generally try to keep our lines below 80 characters see https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md

I personally have a 'ruler' set up in my editor, which shows me exactly where the 80 character limit is set up (you can google that for your editor of choice).

See the above flag `addr` as example how to break the lines.

**Review by sputn1ck — CHANGES_REQUESTED** [review]
Hey @anddddrew thanks for the contribution!

Your PR is now 9 commits 😅 . You could look into how to `squash` your commits into one.

Additionally we like to have our code adhere to our [formatting guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#contribution-standards), see my review for more details.

**Pull request closed without merging**